libs/core/include/CutFlowManager.hpp

```diff
@@ -20,9 +20,11 @@ class CutFlowManager {
   void UpdateCutFlow(std::string cutName, std::string collectionName = "");
   bool HasCut(std::string cutName, std::string collectionName = "");
   std::map<std::string, float> GetCutFlow(std::string collectionName = "");
+  std::map<std::string, float> GetRawEventsCutFlow(std::string collectionName = "");
```

Comment:
Can you remind me why we need those (and isEmpty) to be public? If it's only for the HistogramsFiller, then maybe it would be better to make them private/protected and make the HistogramsFiller a friend class.

Reply:
I also use GetCutFlow in TTAlpsHistogramFiller so they should be public. But the isEmpty I don't see being used anywhere in my code, I don't know if you used it for something maybe?